service/src/main/python/jcarbon/tensorflow/callbacks.py

```diff
@@ -124,6 +124,38 @@ def on_epoch_end(self, epoch, logs=None):
         super().on_epoch_end(epoch, logs)
         self.timestamps[epoch] = pd.DataFrame(data = self.batch_timestamps).assign(
             epoch=epoch)
+            
+# Callback for collecting reports in inference stage. See issue #35: https://github.com/project-jcarbon/jcarbon/issues/35#issue-2551253967.
```

Comment:
Let's move the issue comment up to the JCarbonCallback. You can do `# TODO(https://github.com/project-jcarbon/jcarbon/issues/35#issue-2551253967): Refactor so this can be reused easily`.